WooCommerce/Classes/Tools/In-Person Payments/CardPresentConfigurationLoader.swift

```diff
@@ -1,16 +1,23 @@
 import Foundation
+import Experiments
 import Yosemite
 
 final class CardPresentConfigurationLoader {
-    init(stores: StoresManager = ServiceLocator.stores) {
+    let shouldReturnConfigurationForGB: Bool
+
+    init(stores: StoresManager = ServiceLocator.stores,
+         featureFlagService: FeatureFlagService = ServiceLocator.featureFlagService) {
```

Comment:
This catch my eye, I don't think I'd realized that ServiceLocator is not part of the Yosemite framework 💯